### ISSUE TITLE
Tool descriptions in the main menu

### ISSUE THREAD

**Issue by hadinon** [ncsco/pinemap-dss-help#39]
*From Steven Weaver: Only state what can be found in the tool and avoid providing any additional detail, e.g., “View projected changes in summer temperature” and drop “which could affect…”.
*Clarify the difference between the seedling deployment tools
(MEDIUM PRIORITY)

**Comment by daviswx** [comment]
Per Steven's suggestion, I have updated the climate tool menu descriptions to only state what each tool shows.

For the seedling deployment tools, here's what we currently have:
- View ranges where seedlings from a selected location could be deployed in the future; ideal for nurseries
- View ranges where seedlings could be recruited from for future planting at a given location; ideal for landowners

This seems fairly clear to me, but here's one idea for re-wording and possibly clarifying them:
- View the best regions for future deployment of seedlings from a given location
- View the best seedling source regions for future planting at a given location

Any other ideas or suggestions?  Should we keep the "ideal for nurseries/landowners" part or save that for the landing page?

**Comment by hadinon** [comment]
Sounds good about the menu descriptions. 

In terms of the seedling deployment descriptions, what about using your wording but dropping the "from/at a location"? In other words, you would have: 
- View the best regions for future deployment of seedlings 
- View the best seedling source regions for future planting

I think saving the "ideal for nurseries/landowners" part for the landing page. Does that sound ok?

**Comment by daviswx** [comment]
This is a concise way of putting it, but to me, the confusing part about differentiating these two tools is relating it to your location.  Are you looking to move seed _from_ or _to_ your location?

I wonder if this could be another good question for the beta testers in the optional section on the seedling deployment tools.  Maybe we could ask something like "Which of these sets of descriptions best explains the functionality of the seedling deployment tools, including the difference between them?", and have several different possible ways to word the descriptions.

**Comment by hadinon** [comment]
True. Hmmm before beta testing, do you think we should bring this to Aim 6 first? I'd be happy to shoot off an email asking for some quick feedback by tomorrow. Thoughts?

**Comment by daviswx** [comment]
I'm fine either way, but since most of Aim6 hasn't explicitly reviewed the seedling deployment tools, I worry that the nuance of the differences could be lost on them.  That's why I thought it might be useful to get feedback from the beta testers who reviewed those tools in the last go-around.

**Comment by hadinon** [comment]
Yeah, good point. My only concern is that some of the non-PINEMAP beta testers might not have the expertise to answer that question but I guess they can always skip it! I'll go ahead and add it to the beta testing question doc.

**Comment by daviswx** [comment]
I wonder if we might want to preface the Seedling Deployment Tools section on the beta tester survey with a statement like "These questions are intended as a follow-up to the initial round of beta testing that took place earlier this year.  Please only complete these questions if you participated in that round of testing or are otherwise familiar with the seedling deployment tools."

**Comment by hadinon** [comment]
Yep, sounds good to me

**Comment by daviswx** [comment]
Re: the menu descriptions of the seedling deployment tools...

Only two of the beta testers answered our question evaluating different wording options, but they both said our current wording is most effective among the choices we offered.  Given that, I'm inclined to keep what we already have.

I do think the current descriptions may be a bit wordy and that's probably why we sometimes struggle to remember which tool is which, so maybe we could try to reduce some of the wordiness while keeping the essence of what we have.

I will try to brainstorm some ideas.

**Comment by hadinon** [comment]
Keep current wording until we receive feedback from stakeholders as we roll-out the DSS.
